.github/workflows/helm-check.yaml

```diff
@@ -37,18 +37,15 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.0.0-rc.2
-        with:
-          command: lint
-          config: install/kubernetes/chart-testing.yaml
+      - name: Run helm int
```

Comment:
Typo:

```suggestion
      - name: Run helm lint
```

Reply:
Thanks for pointing out :man_facepalming: